scripts/push-debian.sh

```diff
@@ -34,6 +34,11 @@ if [[ "$(uname)" != "Darwin" && "${BAZEL_BUILD_ARGS}" != *"--config=libc++"* ]];
   BAZEL_BUILD_ARGS="${BAZEL_BUILD_ARGS} --config=libc++"
 fi
 
+# ARCH_SUFFIX allows optionally appending a -{ARCH} suffix to published binaries.
+# For backwards compatibility, Istio skips this for amd64.
+# Note: user provides "arm64"; we expand to "-arm64" for simple usage in script.
+export ARCH_SUFFIX="${ARCH_SUFFIX+-${ARCH_SUFFIX}}"
```

Comment:
Never seen this syntax. What does this do?

Reply:
X="${Y+blah}" means

```
if y is set {
 x=blah
} else
 x=""
}
```

So this just adds a `-` in front of ARCH_SUFFIX unless its empty. To achieve `user provides "arm64"; we expand to "-arm64" for simple usage in script.`